src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -48,7 +48,10 @@ object AgoraBusiness {
   def importResolver(importString: String) : String = {
     val importStringPattern = """^methods\:\/\/(\S+)\.(\S+).(\d+)""".r
     importString match {
-      case importStringPattern(namespace, name, snapshotId) => AgoraBusiness.findSingle(namespace, name, snapshotId.toInt).getOrElse(new AgoraEntity()).payload.getOrElse("")
+      case importStringPattern(namespace, name, snapshotId) =>
+        //hard coded for tasks...we need to ask wdl parser which type we have
```

Comment:
What is going on here? Tasks cannot import other tasks/workflows. But workflows CAN import other workflows. It seems like hardcoding the new AgoraEntity() as a task is fine because we are just using it as something with an empty payload in the subsequent chained calls.